src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,123 @@
+import { useEffect, useState } from "react";
+import { Person } from "../types";
+import { getPeople } from "../api";
+import { Loader } from "./Loader";
+import { Link, useParams } from "react-router-dom";
+import classNames from "classnames";
+
+
+export const PeoplePage = () => {
+
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState<boolean>(true);
+  const [error, setError] = useState<string | null>(null);
+
+  const { personSlug } = useParams();
+
+  const personInfo = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+
+  const personName = (name: string | null) => {
+    return people.find(person => person.name === name);
+  }
```

Comment:
In the 'personName' function, you are comparing the person's name with the name passed as an argument to find the person object. But people can have the same name, so it's better to use a unique identifier like 'slug' or 'id' for comparison. Also, it would be better to rename this function to something like 'findPerson' for better readability and understanding.